std/special/bootstrap.zig

```diff
@@ -43,6 +45,16 @@ nakedcc fn _start() noreturn {
     @noInlineCall(posixCallMainAndExit);
 }
 
+nakedcc fn wasiStart() noreturn {
```

Comment:
Is this specified somewhere? How does this work?

Reply:
No. It starts from regular old `_start`. This was just a placeholder until we got the alloc situation sorted out.